fk/teamjoin.go

```diff
@@ -31,6 +31,10 @@ import (
 )
 
 func teamJoin(teamUUID uuid.UUID) exitCode {
+	if exitCode := ensureNoExistingRequests(teamUUID); exitCode != 0 {
```

Comment:
Doesn't this want to be *after* we've worked out which key they're using? A request to join a key is unique on `(team UUID, fingerprint)`, not on `teamUUID`

Reply:
:) good thinking